resources/lang/pl_PL/tables.yml

```diff
@@ -61,6 +61,8 @@ filter:
   click_to_clear: "Kliknij aby wyczyścić"
   click_to_sync_date_range: "Kliknij aby ustawić jednodniowy zakres"
   click_to_sync_number_range: "Kliknij aby ustawić maks = min"
+  non_null_value: "dowolne"
```

Comment:
nie lepiej "niepuste"?

Reply:
Myślę że słowo "niepuste", jako zresztą nieistniejące w słowniku, nie jest aż takie dobre dla osób nietechnicznych. Jasne że dla nas to jest niepuste, ale dla nich to jest dowolne, w odróżnieniu od braku - tak mi się zdaje. Więc na razie tak zostawię.